### PR TITLE
Some corrections for XTRX

### PR DESCRIPTION
**Pull request by zsoerenm**

### CODE REVIEW

File: src/lms7002m/LMS7002M.cpp [hunk 1]
```diff
@@ -33,7 +33,7 @@ using namespace lime;
 #include "MCU_BD.h"
 
 float_type LMS7002M::gVCO_frequency_table[3][2] = { { 3800e6, 5222e6 }, { 4961e6, 6754e6 }, {6306e6, 7714e6} };
-float_type LMS7002M::gCGEN_VCO_frequencies[2] = {1930e6, 2940e6};
+float_type LMS7002M::gCGEN_VCO_frequencies[2] = {2000e6, 2700e6};
```
Comment (sjkelly) [inline src/lms7002m/LMS7002M.cpp]: Is this wanted?

**Comment by sjkelly** [comment]
LGTM

File: src/lms7002m/LMS7002M.cpp (hunk 1 above)
Comment (zsoerenm) [reply src/lms7002m/LMS7002M.cpp]: Yes, see: https://github.com/xtrx-sdr/liblms7002m/blob/master/liblms7002m.h#L253
and here: https://github.com/JuliaComputing/LMS7002M-driver/blob/master/include/LMS7002M/LMS7002M.h#L47

Comment (zsoerenm) [reply src/lms7002m/LMS7002M.cpp]: Actually, I'm not so sure anymore. It seems to have changed in the past: https://github.com/myriadrf/LimeSuite/commit/ae3bd0fdf5de6826f79dcbd42e5f6287c3f67286
I changed it back to the original value and the VCO can still be tuned.

**Comment by zsoerenm** [comment]
I set some more registers. Some of those are the default of LimeSDR.
Unfortunately, I can still not set the bandwidth of the second channel.
I also observed the following:
If I use this driver (without setting the bandwidth of the second channel), the received signal will be completely zero, even in the digital loopback. Maybe this problem is related to the fact that I cannot set the bandwidth of the second channel?

**Pull request merged**